lms/templates/courseware/course_navigation.html

```diff
@@ -86,21 +86,11 @@
     <ol class="course-tabs">
       % for tab in get_course_tab_list(request, course):
```

Comment:
I think we need to simply things. We should provide ability to override all course tabs instead of each single tab with a separate template file.
we can include a tabs file here like this
`<%static:optional_include_mako file="courseware/tabs.html" >`
Then inside `tabs.html` we should handle rendering of tabs like this
```
 <ol class="course-tabs">
        % for tab in get_course_tab_list(request, course):
...................
...................
...................
</ol>
```